sklearn/callback/tests/_utils.py

```diff
@@ -0,0 +1,107 @@
+# License: BSD 3 clause
+# Authors: the scikit-learn developers
+
+from sklearn.base import BaseEstimator, _fit_context, clone
+from sklearn.callback import BaseCallback
+from sklearn.callback._base import _eval_callbacks_on_fit_iter_end
+from sklearn.utils.parallel import Parallel, delayed
+
+
+class TestingCallback(BaseCallback):
+    def on_fit_begin(self, estimator, *, X=None, y=None):
+        pass
+
+    def on_fit_end(self):
+        pass
```

Comment:
I assume that we can add the `# pragma: no cover` for not covering the `pass` in this class.

Reply:
and the one below.